exonum/src/encoding/segments.rs

```diff
@@ -189,7 +189,7 @@ impl<'a> SegmentField<'a> for RawMessage {
             });
         }
         let actual_size = slice.len() as Offset;
-        let declared_size: Offset = LittleEndian::read_u32(&slice[6..10]);
+        let declared_size: Offset = LittleEndian::read_u32(&slice[5..9]);
```

Comment:
I'm afraid of the new header size. Probably, the alignment will be worse

Reply:
I doubt this will affect performance.

Reply:
Can we benchmark it somehow? It could be possible to come up with an alternative that just permanently sets the `network_id` byte to 0 (with an added benefit that the change would be backward-compatible).

Reply:
On second thought, alignment can be important. Still I don't like `network_id`, so I see several solutions:
- Remove the `network_id` method, but leave header size as 10 bytes by adding a placeholder (`MessageWriter` should add it).
- Adjust other fields, for example change `version` from `u8` to `u16`.